python/lsst/sims/maf/slicers/baseSpatialSlicer.py

```diff
@@ -98,7 +98,7 @@ def _sliceSimData(islice):
                 indices = self.opsimtree.query_ball_point((sx, sy, sz), self.rad)
 
             for key in self.slicePoints.keys():
-                if np.size(self.slicePoints[key]) > 1:
+                if (np.size(self.slicePoints[key]) > 1) & (key is not 'bins') & (key is not 'binCol'):
```

Comment:
You know, I have no idea why this was slicePoint[key][islice] in the first place?